src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -74,6 +74,12 @@ public void before(TestContext context) {
         this.httpBridge = new HttpBridge(this.bridgeConfigProperties);
 
         this.vertx.deployVerticle(this.httpBridge, context.asyncAssertSuccess());
+        try {
```

Comment:
do you really need this sleep? Tests are passing even on Travis without that, are you facing any problems?